test/conformance/ingress/util.go

```diff
@@ -104,16 +114,32 @@ func CreateService(t *testing.T, clients *test.Clients, portName string) (string
 			},
 		},
 	}
+	test.CleanupOnInterrupt(func() {
+		clients.KubeClient.Kube.CoreV1().Services(svc.Namespace).Delete(svc.Name, &metav1.DeleteOptions{})
+	})
 	svc, err = clients.KubeClient.Kube.CoreV1().Services(svc.Namespace).Create(svc)
 	if err != nil {
 		cancel()
 		t.Fatalf("Error creating Service: %v", err)
 	}
 
+	// Wait for the Pod to show up in the Endpoints resource.
```

Comment:
This is required but not enough, I think...

Reply:
Yeah, that's what I'm seeing :(